Description: Use power/control instead of power/level if available
 Move away from the deprecated power/level interface if the newer
 power/control interface is available.
Author: Julien BLACHE <jblache@debian.org>

Index: sane-backends-1.0.22/tools/sane-desc.c
===================================================================
--- sane-backends-1.0.22.orig/tools/sane-desc.c	2011-03-12 11:08:48.417016018 +0100
+++ sane-backends-1.0.22/tools/sane-desc.c	2011-03-12 11:13:36.125015583 +0100
@@ -3543,7 +3543,7 @@
     }
 
   printf("\n# The following rule will disable USB autosuspend for the device\n");
-  printf("ENV{libsane_matched}==\"yes\", RUN+=\"/bin/sh -c 'test -e /sys/$env{DEVPATH}/power/level && echo on > /sys/$env{DEVPATH}/power/level'\"\n");
+  printf("ENV{libsane_matched}==\"yes\", RUN+=\"/bin/sh -c 'if test -e /sys/$env{DEVPATH}/power/control; then echo on > /sys/$env{DEVPATH}/power/control; elif test -e /sys/$env{DEVPATH}/power/level; then echo on > /sys/$env{DEVPATH}/power/level; fi'\"\n");
 
   printf ("\nLABEL=\"libsane_usb_rules_end\"\n\n");
 
